src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0">
```

Comment:
stars--0 it's extra class, add also class `stars`
![image](https://user-images.githubusercontent.com/64437769/142393757-0492e872-0ccd-47d8-9657-c9466faa0bc4.png)